### PR TITLE
[BD-26][EDUCATOR-5808] Add additional data to mfe API responses

### PR DESCRIPTION
**Pull request by viktorrusakov**
Updated proctoring setting API to return additional proctoring service data and also updated exam API to return type of the exam

[YouTrack](https://youtrack.raccoongang.com/issue/OeX_Proctoring-198)
[JIRA](https://openedx.atlassian.net/browse/EDUCATOR-5808)

NOTE: should be merged together with https://github.com/raccoongang/frontend-lib-special-exams/pull/27

### CODE REVIEW

File: edx_proctoring/utils.py [hunk 1]
```diff
@@ -302,11 +302,26 @@ def get_visibility_check_date(course_schedule, usage_key):
...
+        humanized_type = _('a timed exam')
+    elif is_practice_exam:
+        if provider and provider.supports_onboarding:
+            exam_type = 'onboarding'
+            humanized_type = _('an onboarding exam')
+        else:
+            exam_type = 'practice'
+            humanized_type = _('a practice exam')
+    else:
+        exam_type = 'proctored'
```
Comment (idegtiarov) [inline edx_proctoring/utils.py]: Nit: I would use following statement as a default so `else` could be omitted, and rewrite variable definitions as a tuple here and in other parts:
```
exam_type, humanized_type = 'proctored', _('a proctored exam')
```
which will make code become a little bit more laconic.

**Review by idegtiarov — CHANGES_REQUESTED** [review]
one nit please take a look, feel free to ignore

File: edx_proctoring/utils.py (hunk 1 above)
Comment (viktorrusakov) [reply edx_proctoring/utils.py]: It does look better, thanks, updated!

File: edx_proctoring/utils.py [hunk 2]
```diff
@@ -302,11 +302,22 @@ def get_visibility_check_date(course_schedule, usage_key):
...
+        humanized_type = _('a timed exam')
+    elif is_practice_exam:
+        if provider and provider.supports_onboarding:
+            exam_type = 'onboarding'
+            humanized_type = _('an onboarding exam')
+        else:
+            exam_type = 'practice'
+            humanized_type = _('a practice exam')
+
+    return exam_type, humanized_type
```
Comment (idegtiarov) [inline edx_proctoring/utils.py]: I didn't mean exactly that, sorry for misleading.
I'm not a fan of using object by the index it is not clear enough for the review which parameter we are interested in, but we could use named tuples instead. What is your thought on that?

My initial proposal was:
```
    if is_timed_exam:
        exam_type, humanized_type = 'timed', _('a timed exam')
    elif is_practice_exam:        
        if provider and provider.supports_onboarding:
            exam_type, humanized_type = 'onboarding', _('an onboarding exam')
        else:
            exam_type, humanized_type = 'practice', _('a practice exam')

Comment (viktorrusakov) [reply edx_proctoring/utils.py]: Yeah, my bad... At first I actually thought it is strange that you would like to access the tuple by index when we use result of this function, but then I thought maybe it is better, I should have asked you about it first. I will refactor this

**Review by idegtiarov — APPROVED** [review]
Thanks, let's have it landed

**Pull request merged**